content/en/docs/gke/deploy/upgrade.md

```diff
@@ -6,16 +6,30 @@ weight = 5
 
 ## Before you start
 
+To better understand upgrade process, you should read the following sections first:
+
+- [Understanding the deployment process for management cluster](../management-setup#understanding-the-deployment-process)
+- [Understanding the deployment process for Kubeflow cluster](../deploy-cli#understanding-the-deployment-process)
+
+
 This guide assumes the following settings:
 
 - The `${KF_DIR}` environment variable contains the path to
   your Kubeflow application directory, which holds your Kubeflow configuration
   files. For example, `/opt/my-kubeflow/`.
 
   ```bash
-  export KF_DIR=<path to your Kubeflow application directory>
+  KF_DIR=<path to your Kubeflow application directory>
   ```
 
+- The `${KF_NAME}` environment variable contains the name of your Kubeflow deployment.
```

Comment:
Define export commands for all environment variables with placeholders for the values. Update the descriptions to indicate the value to replace and what to replace it with.